ranger/core/loader.py

```diff
@@ -51,12 +52,12 @@ def destroy(self):
 class CopyLoader(Loadable, FileManagerAware):  # pylint: disable=too-many-instance-attributes
     progressbar_supported = True
 
-    def __init__(self, copy_buffer, do_cut=False, overwrite=False, dest=None):
+    def __init__(self, copy_buffer, do_cut=False, resolve_conflict=get_safe_path, dest=None):
```

Comment:
`overwrite`'s not going anywhere. It should take precedence over the new argument.